server/src/main/java/org/elasticsearch/cluster/metadata/Metadata.java

```diff
@@ -2774,8 +2745,6 @@ public static Metadata fromXContent(XContentParser parser) throws IOException {
                         while ((token = parser.nextToken()) != XContentParser.Token.END_OBJECT) {
                             builder.put(IndexMetadata.Builder.fromXContent(parser), false);
                         }
-                    } else if ("hashes_of_consistent_settings".equals(currentFieldName)) {
-                        builder.hashesOfConsistentSettings(parser.mapStrings());
```

Comment:
As far as I can tell the `hashes_of_consistent_settings` field was never serialized toXContent.